plugins/queue/scheduler.js

```diff
@@ -716,6 +716,19 @@ async function unzipArtifacts(executor, config) {
     return enq;
 }
 
+/**
+ * Pushes a message to unzip artifacts
+ * @async  sendWebhook
+ * @param  {Object} executor
+ * @param  {Object} webhookConfig               webhookConfiguration
+ * @return {Promise}
+ */
+async function sendWebhook(executor, webhookConfig) {
+    await executor.connect();
+
+    return executor.redisBreaker.runCommand('rpush', executor.webhookTable, JSON.stringify(webhookConfig));
```

Comment:
`webhookConfig` has no fields that can be used for identifiers. So I use list instead of hash.

Reply:
[ASK]
What values are saved in `redis`?
`key: ??? value: hookId`

Reply:
The value is `JSON.stringify(webhookConfig)` and it does not have key. I use list as data type and the keys are not required.
[Redis Data Types](https://redis.io/topics/data-types)

Reply:
I was going to use `hookId` as key, but that would not work in gitlab. So I chose a list as the data structure, which doesn't need a key.
[hookId of gitlab](https://github.com/screwdriver-cd/scm-gitlab/blob/e22370a28b41a8236866996ca31a96e978b8b14c/index.js#L369)

Reply:
OK, I understand.